content/en/blog/2023/dlb-connection-balance/index.md

```diff
@@ -0,0 +1,305 @@
+    - Distributes high bandwidth flows across many cores without loss of packet order
+    - Better determinism and avoids excessive queuing latencies
+    - Uses less IO memory footprint and saves DDR Bandwidth
+- Priority queuing (up to 8 levels) —allows for QOS
+    - Lower latency for traffic that is latency sensitive
+    - Optional delay measurements in the packets
+- Scalability
+    - Allows dynamic sizing of applications, seamless scale up/down.
+    - Power aware; application can drop workers to lower power state in cases of lighter load.
+
+There are three types of load balancing queues:
+
+- Unordered: For multiple producers and consumers. The order of tasks is not important, and each task is assigned to the processor core with the lowest current load.
+- Ordered: For multiple producers and consumers where the order of tasks is important. When multiple tasks are processed by multiple processor cores, they must be rearranged in the original order.
+- Atomic: For multiple producers and consumers, where tasks are grouped according to certain rules. These tasks are processed using the same set of resources and the order of tasks within the same group is important.
+
+An ingress gateway is expected to process as much data as possible as quickly as possible, so Intel DLB connection load balance uses an unordered queue.
+
+## How to use Intel DLB connection load balance in Istio
+
+With the 1.17 release, Istio supports Intel DLB connection load balance officially.
+
+The following steps show how to use Intel DLB connection load balance in an Istio [Ingress Gateway](../../../docs/tasks/traffic-management/ingress/ingress-control/) in an SPR (Sapphire Rapids) machine and Kubernetes is ready.
+
+### Step 1 Prepare DLB environment
+
+Install the Intel DLB driver by following [the instructions on the Intel DLB driver official site](https://www.intel.com/content/www/us/en/download/686372/intel-dynamic-load-balancer.html).
+
+Install the Intel DLB device plugin with the following command:
+
+{{< text bash >}}
+$ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/dlb_plugin?ref=v0.26.0
+{{< /text >}}
+
+For more details about the Intel DLB device plugin, please refer to [Intel DLB device plugin homepage](https://www.envoyproxy.io/docs/envoy/latest/configuration/other_features/dlb#config-connection-balance-dlb).
+
+You can check the Intel DLB device resource:
+
+{{< text bash >}}
+$ kubectl describe nodes | grep dlb.intel.com/pf
+  dlb.intel.com/pf:   2
+  dlb.intel.com/pf:   2
+...
+{{< /text >}}
+
+### Step 2 Download Istio
+
+Now the latest version is 1.17.2, let’s download the installation:
+
+{{< text bash >}}
+$ curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.17.2 TARGET_ARCH=x86_64 sh -
+$ cd istio-1.17.2
+$ export PATH=$PWD/bin:$PATH
+{{< /text >}}
+
+**TIPS: All following actions will be done under this directory.**
```

Comment:
I suspect we can use the normal Istio callout here: https://istio.io/latest/docs/releases/contribute/shortcodes/#callouts

Reply:
Good idea!